docs/Architecture Decision Record/009-git-workflow.md

```diff
@@ -49,8 +50,9 @@ A contributor to the TDP project would always use the following steps to propose
     * removes `raft-review` label
     * adds `qasp-review` label
     * assigns the Government as the reviewer:
-        * For backend development work assign `alexsoble`, `adpennington` and `lfrohlich`
-        * For frontend work, assign the same as backend work but also include `iamjolly` and `ttran-hub` for a11y review
+        * For backend and frontend development work assign `adpennington` and `alexsoble`
```

Comment:
Not related but just noting we'll want to remove me from this line during offboarding ... need to start building up an offboarding checklist! 😢